lib/servers/abstractservers/script_scanner/script_scanner.py

```diff
@@ -90,44 +88,45 @@ def load_scripts(self):
                 else:
                     self.script_parameters[name] = script_class_parameters(name, cls, parameters)
 
-    @setting(0, "Get Available Scripts", returns = '*s')
+    @setting(0, "get_available_scripts", returns='*s')
```

Comment:
Will clients running the script_scanner server still be able to use the same method name with this change?